Http/controllers/students/index.php

```diff
@@ -5,10 +5,20 @@
 
 $db = App::resolve(Database::class);
 
-$students = $db->query('SELECT * FROM students')->get();
+$users = $db->query('SELECT * FROM users')->get();
+
+$userStudents = $db->query('SELECT 
+                        students.fname AS student, 
+                        GROUP_CONCAT(users.email) AS mentor,
+                        students.id AS student_id
+                        FROM users_students
+                        JOIN students ON users_students.student_id = students.id
+                        JOIN users ON users_students.user_id = users.id
```

Comment:
```suggestion
                        FROM students
                        LEFT JOIN users_students ON students.id = users_students.student_id
                        LEFT JOIN users ON users_students.user_id = users.id
```
My previous comment is not fixed. It was still showing only relations. 
After applying my fix, it will look like this:
![image](https://github.com/user-attachments/assets/cae80992-5994-4e30-95f7-33561717ccba)